torchx/schedulers/ray_scheduler.py

```diff
@@ -187,17 +194,23 @@ def schedule(self, dryrun_info: AppDryRunInfo[RayJob]) -> str:
                     copy2(script,dirpath)
 
         # 2. Copy Ray driver utilities
-        copy2("./ray", dirpath)
+        current_directory = os.path.dirname(os.path.abspath(__file__))
+        copy2(os.path.join(current_directory,"ray", "ray_driver.py"), dirpath)
+        copy2(os.path.join(current_directory,"ray", "ray_common.py"), dirpath)
+
 
         # 3. Copy actor.json
-        actor_path = find_file("actor", "/tmp")
-        copy2(actor_path, dirpath)
+        # actor_path = find_file("actor", "/tmp")
```

Comment:
This part is not needed right? You are already copying `actors.json` in `serialize()`.